Vagrantfile

```diff
@@ -2,10 +2,19 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "lucid64"
-  config.vm.box_url = "http://files.vagrantup.com/lucid64.box"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
I don't use vagrant for my testing so I can't validate the old scripts. It would be nice to back the change out and put it in it's own pr if you want to change them.